crates/api_common/src/person.rs

```diff
@@ -99,6 +99,7 @@ pub struct GetPersonDetails {
   pub limit: Option<i64>,
   pub community_id: Option<CommunityId>,
   pub saved_only: Option<bool>,
+  pub show_deleted_and_removed: Option<bool>,
```

Comment:
@dessalines Are you planning to use this new param in lemmy-ui? If not i would simply remove it, to not make the api unnecessarily complicated.

Reply:
Hrm... yeah you're probably right. The only case I can think of, is a mod wanting to restore comments... but they should be doing that from the comment threads anyway where they can see context. 

I'll do a PR shortly.